features/personas/ChangePassword.feature

```diff
@@ -0,0 +1,12 @@
+@javascript @changePassword
+Feature: Verify than an User allowed to change password can change his password
```

Comment:
Verify that

Reply:
Indeed. 😄

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1764/commits/1fd9e7af0187b5bb29318b3721eb35c27d4f236b.